sidebars.js

```diff
@@ -189,6 +189,7 @@ module.exports = {
             'latest/development/patches',
             'latest/development/source-code-directory-structure',
             'latest/development/testing',
+            'latest/development/api-history-migration-guide',
```

Comment:
nit: these are otherwise sorted alphabetically, do we want to do the same with this one?